test/k23si/K23SITest.cpp

```diff
@@ -90,15 +90,11 @@ class K23SITest {
                     },
                     .rangeEnds{}
                 };
-                return RPC().callRPC<dto::CollectionCreateRequest, dto::CollectionCreateResponse>
-                        (dto::Verbs::CPO_COLLECTION_CREATE, request, *_cpoEndpoint, 1s);
+                return _cpo_client.createAndWaitForCollection(Deadline<>(_createWaitTime()), std::move(request.metadata), std::move(request.rangeEnds));
             })
-            .then([](auto&& response) {
+            .then([](Status&& status) {
```

Comment:
Seems like this .then block is a no-op now and the K2EXPECT call can be moved into the next one.